freud/order.pyx

```diff
@@ -272,6 +286,22 @@ cdef class Hexatic(_PairCompute):
     def compute(self, system, neighbors=None):
         R"""Calculates the hexatic order parameter.
 
+        Example::
+
+            >>> # Assuming user has defined k and box, points/positions,
+            >>> # and system(box, points) have already been defined
+            >>> # Compute the K-atic order for the 2 D system
+            >>> # Recommend visualizing data with the particle_property below
+            >>> import freud
+            >>> from ovito.data import *
+            >>> import numpy as np
+            >>> order_param = freud.order.Hexatic(k)
+            >>> order_param.compute(system)
+            >>> psi_k = np.copy(order_param.particle_order)
+            >>> psi_k -= np.mean(psi_k)
```

Comment:
I'm not sure these lines are necessary. If we think the `Hexatic` would be beneficial to show people how to access the final order parameters, it may be more useful just to print the values, like in these `hoomd` examples [[x](https://hoomd-blue.readthedocs.io/en/stable/module-hoomd-data.html)]:
'''python
>>> print(order_param.particle_order)
array([ 0.2119954 -0.5681961j ,  0.18827692-0.4118587j ,
       -0.13218744-0.6692502j , ..., -0.4630672 -0.15254533j,
        0.36158592+0.08465159j,  0.15578252-0.44301566j], dtype=complex64)
'''